apps/backend/src/modules/onboarding/onboarding-question.model.ts

```diff
@@ -0,0 +1,33 @@
+import { Model, type RelationMappings } from "objection";
+
+import {
+	AbstractModel,
+	DatabaseTableName,
+} from "~/libs/modules/database/database.js";
+
+import { OnboardingAnswerModel } from "./onboarding-answer.model.js";
+
+class OnboardingQuestionModel extends AbstractModel {
+	public answers!: OnboardingAnswerModel[];
+
+	public label!: string;
+
+	static get relationMappings(): RelationMappings {
+		return {
+			answers: {
+				join: {
+					from: `${DatabaseTableName.ONBOARDING_QUESTIONS}.id`,
+					to: `${DatabaseTableName.ONBOARDING_ANSWERS}.questionId`,
```

Comment:
I noticed in some places you used `questionId`, in other places `question_id`. Make sure we have camelCase everywhere